packages/vite-plugin-checker/src/checkers/typescript/main.ts

```diff
@@ -121,13 +122,21 @@ const createDiagnostic: CreateDiagnostic<'typescript'> = (pluginConfig) => {
 
         ts.createSolutionBuilderWithWatch(host, [configFile], {}).build()
       } else {
+        // onchange:add debounce
+        let debounceReportWatchStatusChanged = reportWatchStatusChanged
```

Comment:
I'm not whether `tsc --watch` has a built-in debounce tragedy already. Have you ever experienced race condition when using `tsc`?

Reply:
if eslint `lint on save` in editor,it will also trigger tsc multiple times. so i add debounce in typescript.
<img width="1492" alt="image" src="https://user-images.githubusercontent.com/15699844/216879820-2e22b3a9-2d90-459e-8f35-efa11e260cbe.png">